src/ocean/util/UnixSocketCommandInterface.d

```diff
@@ -0,0 +1,161 @@
+/*******************************************************************************
```

Comment:
I question the choice of `util` for this module. As it's inextricably connected with `ocean.net.server.unix.UnixListener`, wouldn't it make more sense to put it in the same package? (I know it's not actually related to network I/O, but the same goes for loads of the stuff in `ocean.net.http`, for example.)

Reply:
It didn't feel all right for me to be in `ocean.net.*`, since it does not have much to do with the network itself - this just presents itself as way to add and remove the handlers and run them. And it's really the utility module IMHO. I somehow feel the difference between `http` and this - while the former is full blown http server, this is just a small utility.

I'm open for any other proposals, though, as I don't feel strongly about this.